### DISCUSSION TITLE
Snap Panel Size

### DISCUSSION THREAD

**Discussion by emmtre** [rxhanson/RectanglePro-Community#321] [General]
Is there a hidden setting to change the Snap Panel size (height) via a Terminal command? I looked in the preference plist but I couldn't find anything related more then the Snap Panel actions.

**Comment by rxhanson** [comment]
Thanks for asking. Not at this time. The main thing holding back this functionality is that I'd have to go through and create higher resolution versions of all the icons (they're not vector based). Both of these tasks are on my backlog, it's just a matter of getting to it.

**Reply by emmtre** [reply]
Many thanks for the quick reply. I would have guessed it was something like that. No problems at all and I was just curious. Keep up the good work!
